pkg/test/resourcefixture/testdata/basic/storage/v1beta1/storagebucket/update.yaml

```diff
@@ -23,3 +23,5 @@ spec:
   versioning:
     enabled: true
   lifecycleRule: []
+  softDeletePolicy:
+    retentionDurationSeconds: 0
```

Comment:
I ran the test locally and found that `0` was not used in the update, and the value kept to be `604800`. Could you verify that this is a valid update; or whether setting the value to `0` is actually effective? AFAIK, we don't support setting a primitive-typed field to its empty value.

Reply:
I checked again and actually it worked.